jvb/src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -179,7 +179,7 @@
      */
     private ScheduledFuture<?> updateLastNEndpointsFuture;
 
-    private final EndpointConnectionStatusMonitor epConnectionStatusMonitor;
+    private EndpointConnectionStatusMonitor epConnectionStatusMonitor;
```

Comment:
Can this still be `final` if we explicitly initialize it to `null` in the `!enableLogging` case?  (Would it be worth it?)

Reply:
Looks like it can.  It probably doesn't make a huge difference but `final` is always nice when we can.